spec/somemoji/emoji_spec.rb

```diff
@@ -3,39 +3,42 @@
     described_class.new(
       aliases: [],
       ascii_arts: [],
-      category: "symbols",
-      code: "100",
+      category: "nature",
+      code: "sunny",
       code_points: [
-        "1F4AF"
+        "2600",
+        "fe0f"
       ],
       keywords: [
-        "100",
-        "a",
-        "exam",
-        "numbers",
-        "parties",
-        "percent",
-        "perfect",
-        "plus",
-        "quiz",
-        "school",
-        "score",
-        "symbol",
-        "test",
-        "win",
-        "wow"
+        "brightness",
+        "day",
+        "hot",
+        "morning",
+        "sky",
+        "sun",
+        "weather"
       ],
-      name: "hundred points symbol"
+      "name": "black sun with rays"
```

Comment:
Why did you change the Hash key syntax from `name:` to `"name":` ?

Reply:
It's my mistake :pray: